crypto/key_manager.go

```diff
@@ -84,8 +79,8 @@ func parsePrivateKey(key []byte) (crypto.PrivateKey, error) {
 }
 
 // NewFromPrivatePEM returns key manager for a password protected PEM object.
```

Comment:
For example, `NewFromPrivateKey()` says "The key **may** be protected by a password".